deps/build.jl

```diff
@@ -78,4 +111,11 @@ copy_config("logo-32x32.png", juliakspec)
 copy_config("logo-64x64.png", juliakspec)
 
 eprintln("Installing julia kernelspec $spec_name")
-run(`$jupyter kernelspec install --replace --user $juliakspec`)
+run(`$jupyter-kernelspec install --replace --user $juliakspec`)
```

Comment:
I suppose you are doing this because of jupyter/notebook#448.

@minrk, will Jupyter always install `jupyter-kernelspec` into the same PATH as `jupyter`?  Are `jupyter-kernelspec` and `jupyter-notebook`  part of the public "API"?

I'm worried that at some point Jupyter may install the subsidiary programs like `jupyter-notebook` into a hidden internal path, or eliminate them entirely.